python/mqt/syrec/simulation_view/dialogs/simulation_run_json_import_dialog.py

```
@@ -0,0 +1,244 @@
+        self._update_progress_text_with_batch_info(
+            len(generated_simulation_run_models), batch_generation_duration_in_seconds
+        )
+
+        if self.shared_simulation_runs_model is None:
+            log_error_to_console("Shared simulation runs model was not initialized during handling of batch!")
+            self._handle_non_recoverable_error(None)
+            return
+
+        try:
+            self.shared_simulation_runs_model.add_simulation_run_models(generated_simulation_run_models)
+        except Exception as sim_run_model_err:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)